03/app.js

```diff
@@ -1,3 +1,15 @@
 export default function randomNumber(min, max) {
-
-}
\ No newline at end of file
+  if (min > max) {
+    throw new Error("Min cannot be greater than max!");
+  }
+  if (typeof max !== "number") {
```

Comment:
Tutaj też warto zwrócić uwagę na to, że `NaN` to też `number`, ale raczej nie chcemy, aby taka liczba "przechodziła"